_posts/2020-04-05-firestore-index-terraform.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Importing firestore index to terraform"
+layout: post
+date: 2020-04-05 12:43
+headerImage: false
+tag:
+- terraform
+- firestore
+- google cloud
+category: blog
+author: kosyanyanwu
+description: Import existing firestore index to be managed by terraform
+---
+## Introduction
+Google Cloud Firestore ensures query performance by requiring an index for every query. Index creation takes a few minutes, depending on how much data needs to be updated. The more documents that match the fields being indexed, the longer creation takes. Also, for each unique collection ID, you can have only one index build in progress. Multiple index builds on the same collection ID complete sequentially.
```

Comment:
Add links so a reader can click on it to access Firestore

Reply:
thanks